server/api/models/Class.js

```diff
@@ -4,13 +4,18 @@ module.exports = (sequelize, DataTypes) => {
     {
       class: {
         type: DataTypes.TEXT,
-        allowNull: false,
+        notEmpty: true,
+        notNull: true,
         unique: true,
         set(value) {
           this.setDataValue('class', value.toLowerCase());
         }
       },
-      seniors: { type: DataTypes.BOOLEAN, allowNull: false, defaultValue: 0 }
+      seniors: {
+        type: DataTypes.BOOLEAN,
+        notEmpty: true,
+        notNull: true
+      }
```

Comment:
Currently, by not allowing the Senior field to be unique creates an integrity issue.  The seniors field is a Boolean which if true, indicates the current class is a senior. Because the value is not unique work has to be done to ensure there is only one senior class.  I am looking into how to work on this, but ultimately I believe the only way ensure only one class is senior is to run a check for current seniors before adding or changing a senior class.